modules/prometheus-exporter_oracle/conf/07-sessions.yaml

```diff
@@ -0,0 +1,23 @@
+module: oracle
+name: "limit for sessions"
+id: sessions_limits
+
+transformation: true
+aggregation: ".mean(by=['job'])"
+
+signals:
+  signal:
+    metric: oracledb_Sessions_limits_value 
+    rollup: latest
+
+rules:
+  warning:
+    threshold: 85 
+    comparator: ">"
+    lasting_duration: "5m"
+    description: "warning is reached"
```

Comment:
I don't find the metric `Sessions_limits_value` on https://github.com/iamseth/oracledb_exporter again so I am not fully sure of the meaning.

However according to the thresholds you set I suppose this is a percentage of connections over the limit.
in this case the limit is not reached so the description is false.

either keep the default description which seems enough:
> number of sessions compared to limit is too high (X > 85)
or use a custom one which make sens like: `is too close`.